storage/engine/rocksdb.go

```diff
@@ -543,14 +384,14 @@ func (r *rocksDBSnapshot) Defer(func()) {
 
 type rocksDBBatch struct {
 	parent *RocksDB
-	batch  *C.DBBatch
+	batch  *C.DBEngine
```

Comment:
Could this just directly be a C.DBBatch?  I don't see the scenario where this is not a Batch.